src/Benchmarks.ClientJob/Benchmarks.ClientJob.csproj

```diff
@@ -4,12 +4,12 @@
 
   <PropertyGroup>
     <Description>Class to transfer data to/from the benchmark client.</Description>
-    <TargetFramework>netstandard1.0</TargetFramework>
```

Comment:
This is an internal DLL shared by different components of BenchmarkAutomation.  So it doesn't really matter what we choose here.

However, following the spirit of the changes, I do think this should stay `netstandard1.0` since it doesn't depend on anything higher.  But, nothing should break if this is moved to `netcoreapp2.0` either.